cmd/ui/src/store.ts

```diff
@@ -77,19 +85,30 @@ if (SAMLToken !== undefined) {
     Cookies.remove('token');
 }
 
-export const store = configureStore({
-    reducer: rootReducer,
-    preloadedState: initialState,
-    middleware: (getDefaultMiddleware) => {
-        return [...getDefaultMiddleware({ serializableCheck: false }), sagaMiddleware];
-    },
-});
+const initStore = (preloadedState: PreloadedState<RootState>) => {
+    return configureStore({
+        reducer: rootReducer,
+        preloadedState: preloadedState,
+        middleware: (getDefaultMiddleware) => {
+            return [...getDefaultMiddleware({ serializableCheck: false }), sagaMiddleware];
+        },
+    });
+};
+
+export const store = initStore(initialState);
 
 // Persist the session token in local storage
 store.subscribe(
     throttle(() => {
+        const state = store.getState();
         saveState({
-            auth: { sessionToken: store.getState().auth.sessionToken },
+            auth: { sessionToken: state.auth.sessionToken },
+            global: {
+                view: {
+                    darkMode: state.global.view.darkMode,
+                    notifications: [],
```

Comment:
same here

Reply:
See reasoning in previous comment. This was quick and easy while avoiding further refactors.